<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1181</width>
     <height>939</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>905</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>실행</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="macroTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>90000</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>명령어</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>값</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>동작</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>동작2</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="btnConfigInsert_4">
            <property name="maximumSize">
             <size>
              <width>297</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>명령어 삽입</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnConfigAdd_4">
            <property name="maximumSize">
             <size>
              <width>297</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>명령어 추가</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnConfigRemove_4">
            <property name="maximumSize">
             <size>
              <width>297</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>명령어 삭제</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lbFileList">
          <property name="maximumSize">
           <size>
            <width>294</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>파일 목록</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lsFiles">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>120</height>
           </size>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideLeft</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="gridSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="viewMode">
           <enum>QListView::ListMode</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="btnCapture">
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string> 캡쳐</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDeleteFile">
            <property name="maximumSize">
             <size>
              <width>94</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>삭제</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDeleteAllFiles">
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>전체 삭제</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbPreview">
        <property name="minimumSize">
         <size>
          <width>480</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0">
      <item>
       <widget class="QPushButton" name="btnToPdf">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>PDF로 저장</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="maximumSize">
         <size>
          <width>664</width>
          <height>24</height>
         </size>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>시작</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>정지 (F2)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="actionOpen_O"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit_q"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_O">
   <property name="text">
    <string>열기(&amp;o)</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>저장(&amp;s)</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>다른 이름으로 저장(&amp;a)</string>
   </property>
  </action>
  <action name="actionExit_q">
   <property name="text">
    <string>종료(&amp;q)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
